src/java/org/apache/cassandra/io/compress/ICompressorFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.io.compress;
+
+import java.util.Map;
+import com.google.common.collect.ImmutableMap;
+
+public interface ICompressorFactory
+{
+    public static final ImmutableMap<String, String> COMPRESSOR_NAME_MAP = ImmutableMap.of(
```

Comment:
this is unused and can be removed, also not correct as such, we would need to be way more robust than this in general.